src/Symfony/Component/Yaml/Tests/ParserTest.php

```diff
@@ -1898,6 +1898,30 @@ public function taggedValuesProvider()
                 [new TaggedValue('foo', 'bar')],
                 '[ !foo bar ]',
             ],
+            'with-comments' => [
+                [
+                    [new TaggedValue('foo', ['foo', 'baz'])],
+                ],
+                <<<YAML
+- [!foo [
+    foo,
+    baz,
+    #bar
+  ]]
+YAML
+            ],
+            'with-comments-trailing-comma' => [
```

Comment:
I fail to see how this test case differs from the previous one.

Reply:
Oops you're right. The 'with-comments' was meant to be without a comma after the 2nd array element.